docs/Nutrition1.md

```diff
@@ -2,57 +2,54 @@
 
 ## Overview
 
-## Step 1
-#Meal plan log in page
+## Step 1 
+- Access the Admin UI with permissions (Username & Password) assigned as a meal designer.
 
-- Admin shoud be able to access the log in page using the assigned user name and password after inputting the web address [https://goodmealplan.com/](https://goodmealplan.com/) .
+![ADMIN UI login](https://github.com/yemilad/mealplanner/assets/165817613/7552603c-6a27-4917-bb0b-07627941d69e)
```

Comment:
Sorry, one more change.  Instead of the full URL to the image, just use the image name.  e.g:

![ADMIN UI Login](vjoJciS.jpg)

Reply:
And check in the images into the docs/ directory alongside the .md file(s)

Reply:
The corrections have been made.